### PR TITLE
Added express rest api

### LINKED ISSUE #14: Write an express rest api for basic CRUD operations.
The projects requires a rest api for the frontend to make requests. Writing a rest api will allow CRUD operations to be performed on the blogs.
Will write a basic rest api for CRUD and later on the rest api can be changed according to the need when authentication and other stuff is implemented. 

- [x] DCP21

### PR DESCRIPTION
**Pull request by sanyogchangmai**
## Related Issue

Closes: #14 

## Changes made

Wrote an express rest api for basic CRUD operations.

## Saving blog
savePost
save

## Displaying all blogs
showAllPost

## Update blog
updatePost
update

## Delete blog
deletePost
delete

## Viewing particular blog
viewParticular

### CODE REVIEW

File: backend/.env.sample [hunk 1]
```diff
@@ -1 +0,0 @@
-DATABASE="mongodb+srv://<username>:<password>@<clustername>/<database>?retryWrites=true&w=majority" or your local mongo atlas url
```
Comment (dhwaj1902) [inline backend/.env.sample:1]: Why you delete this?

Comment (dhwaj1902) [reply backend/.env.sample:1]: Rest looks good to me!✌

Comment (sanyogchangmai) [reply backend/.env.sample:1]: I just renamed the file name to .env and replaced the above mentioned line with my Mongo DB URI. I shouldnt have deleted that actually , I realized it just now.

Comment (sanyogchangmai) [reply backend/.env.sample:1]: If you want I can create that file again and will not include that file in gitignore.

Comment (dhwaj1902) [reply backend/.env.sample:1]: ok ok no problem, I will merge this. Looks good to me.

**Pull request merged**